tests/unit/accounts/test_views.py

```diff
@@ -47,7 +48,11 @@ def test_too_many_failed_logins(self, pyramid_request):
 
         assert resp.status == "429 Too Many Failed Login Attempts"
         assert resp.detail == (
-            "There have been too many unsuccessful login attempts. Try again later."
+            "There have been too many unsuccessful login attempts. "
+            "You have been locked out for {0}. "
+            "Please try again later.".format(
+                humanize.naturaldelta(exc.resets_in.total_seconds())
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏make the return value explicit here instead of using humanize to calculate it, replacing this line with what we expect humanize to emit, like “5 minutes”.
This reduces the chance that a change to humanize’s behavior pass unchecked into production, since the intent is expressed more clearly in the test case.